api/v1alpha1/common_test.go

```diff
@@ -778,7 +807,8 @@ func TestIsCondSame(t *testing.T) {
 	}
 }
 
-func TestEqual(t *testing.T) {
+// TestComponentVersionEqual for ComponentVersion.Equal
```

Comment:
I didn't find exact documentation or recommendations, but it seems that, there will no comments to the TestXXX function unless there is something about the function that needs extra attention.

An example of go source code: https://github.com/golang/go/blob/023668638298d0abf81090dd44464d3c8b724aa7/src/io/io_test.go#L83
An example of kubernetes: https://github.com/kubernetes/kubernetes/blob/2c6c4566eff972d6c1320b5f8ad795f88c822d09/staging/src/k8s.io/apiserver/pkg/cel/types_test.go#L23
An example of go source code with comment:
https://github.com/golang/go/blob/023668638298d0abf81090dd44464d3c8b724aa7/src/io/io_test.go#L104

Reply:
Yes I agree that those comments can be deleted. I personally use those comments to go to the corresponding functions (that are being tested). However, because writing those can take some time, can we first leave those comments there, and delete them later if we decided to?